int32 face_count
int32[] tracking_id
int32[] x
int32[] y
int32[] size
int32[] conf 
